templates/operations.ts.hbs

```diff
@@ -121,7 +121,8 @@
         {{/if}}
       }>,
       rawResponse?: boolean,
-    ): Promise<Response | {{getReturnTypeFromOperation this}}> {
+      returnTypeParam?: returnType
```

Comment:
```suggestion
      returnType?: MethodReturnType
```
We already know that it's a parameter, because that's the context in which we are using it. 😉